tests/appsec/rasp/test_lfi.py

```diff
@@ -145,7 +145,6 @@ def setup_lfi_stack_trace(self):
         self.r = weblog.get("/rasp/lfi", params={"file": "../etc/passwd"})
 
     def test_lfi_stack_trace(self):
-        assert self.r.status_code == 403
```

Comment:
Yes it's not mandatory to test it here but isn't "nice to have"?

Reply:
yes but as explained in the PR description, for libraries and frameworks that don't support blocking but still support stack traces, we can't enable this test. So to still test the stacktraces without blocking, i propose to remove them. Blocking is testing in plenty of other tests